src/Utils/Helpers.ts

```diff
@@ -32,9 +32,10 @@ export function obtainWorkspaceRoot(): string {
     throw new Error('Need workspace');
   }
 
-  // TODO support active workspace from multiple workspace
+  // TODO support active workspace among the multiple workspaceFolders
+  // TODO support multi-root workspace
   if (workspaceFolders.length > 1) {
-    Balloon.info('Warning: only first Workspace is supported');
+    Balloon.info('Warning: Only the first workspace directory will be recognized by ONE-vscode');
```

Comment:
Can you check https://github.com/Samsung/ONE-vscode/issues/311 ?
However, using `ONE` instead of `ONE-vscode` seems not actually correct...
How about `Warning: Only the first workspace directory is currently supported` ?

Reply:
Note: we need `Ballon.warning()` 😹

Reply:
FYI, this PR actually don't include changes about `Helpers.ts`, let's edit only the message contents this time which is relevant to this PR.

Reply:
AFAIK, The balloon shows the source :wink: 
![image](https://user-images.githubusercontent.com/50489513/168773190-3df508d9-cdc5-4d31-9737-16c3c0964197.png)

Reply:
Oh, sure.

Reply:
https://github.com/Samsung/ONE-vscode/pull/651#discussion_r874560015
`Source: one-vscode` seems interesting. T.T